app/src/main/res/layout/contributors_activity.xml

```diff
@@ -69,6 +96,7 @@
                             android:gravity="center"
                             android:text="@string/contributors_headline"
                             android:textAppearance="@style/NavView.TextView.Medium" />
+
```

Comment:
Same here, no blank lines before closing tags.

Reply:
The best idea is to just remove every blank line in the entire file, then let AS format it for you.